.github/workflows/tests.yml

```diff
@@ -15,7 +15,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        test: [base, many, aux, topo, modules, dhcp]
+        test: [aux]
```

Comment:
I assume this is debugging?

Reply:
This was branched off my master which had this change. I'll clean this up when I move this out of draft.